### PR TITLE
Add Leapp Copr repo for Fedora to docs

### PR DESCRIPTION
**Pull request by bocekm**
Related: https://github.com/oamg/leapp-repository/issues/124

### CODE REVIEW

File: docs/source/devenv-install.md [hunk 1]
```diff
@@ -7,23 +7,28 @@ the RPM packages provided by the [Copr](https://copr.fedorainfracloud.org/coprs/
...
-    yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/g/leapp/leapp-devel/repo/epel-7/group_leapp-leapp-devel-epel-7.repo
-```
+* On CentOS/RHEL:
+  ```shell
+  $ yum install -y yum-utils
+  $ yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/g/leapp/leapp-devel/repo/epel-7/group_leapp-leapp-devel-epel-7.repo
+  ```
+* On Fedora (replace `30` in the URL below if you're on another version of Fedora, but make sure such a repository exists on the Copr):
+  ```shell
+  $ dnf config-manager --add-repo https://copr.fedorainfracloud.org/coprs/g/leapp/leapp-devel/repo/fedora-30/group_leapp-leapp-devel-fedora-30.repo
```
Comment (pirat89) [inline docs/source/devenv-install.md]: I think that this is not the best approach. I prefer way described directly in the copr:
```shell
 # dnf copr enable @leapp/leapp-devel
```

With that command, you don't need to point to the specific version of fedora.

File: docs/source/devenv-install.md [hunk 2]
```diff
@@ -7,23 +7,28 @@ the RPM packages provided by the [Copr](https://copr.fedorainfracloud.org/coprs/
 build system, which automatically builds packages with every commit merged into master.
 Packages are built for EPEL and Fedora.
 
-```shell
-    yum install -y yum-utils
-    yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/g/leapp/leapp-devel/repo/epel-7/group_leapp-leapp-devel-epel-7.repo
-```
+* On CentOS/RHEL:
+  ```shell
+  $ yum install -y yum-utils
```
Comment (pirat89) [inline docs/source/devenv-install.md]: In all cases, you are supposed to be root, not a user.
```suggestion
  # yum install -y yum-utils
```

Please fix the others as well.

File: docs/source/devenv-install.md [hunk 3]
```diff
@@ -32,44 +37,45 @@ To keep your environment clean, use a virtualenv.
...
-	$ . tut/bin/activate
+$ cd ~/devel
+$ virtualenv -p /usr/bin/python2.7 tut
+$ . tut/bin/activate
 ```
 
 Then, install the framework by using the pip package management system:
 ```shell
-	$ pip install git+https://github.com/leapp-to/leapp
+$ pip install git+https://github.com/leapp-to/leapp
```
Comment (pirat89) [inline docs/source/devenv-install.md]: Isn't that old link? :smiling_imp:

**Comment by bocekm** [comment]
Updated per review comments.

File: docs/source/devenv-install.md (hunk 3 above)
Comment (bocekm) [reply docs/source/devenv-install.md]: It's being redirected, but you're right.

File: docs/source/devenv-install.md (hunk 2 above)
Comment (bocekm) [reply docs/source/devenv-install.md]: I fixed it for a few other command where it makes sense.

**Pull request merged**